onadata/apps/logger/data_migration/decisioner.py

```diff
@@ -38,30 +43,47 @@ def __init__(self, xforms_comparator, **data):
 
     @property
     def fields_changes(self):
+        """Return migration changes. Spec:
+
+        :list new_fields: new fields in migration
+        :list removed_fields: removed fields in migration
+        :dict modifications: describes variable renaming decisions. Format:
+            {<prev_name>: <new_name>}
+        """
         return {
-            'new_fields': self.new_fields,
-            'removed_fields': self.removed_fields,
-            'modified_fields': self.modifications,
+            NEW_FIELDS_KEY: self.new_fields,
+            RM_FIELDS_KEY: self.removed_fields,
+            MOD_FIELDS_KEY: self.modifications,
         }
 
     @staticmethod
-    def reverse_changes(changes):
+    def construct_changes(new=None, removed=None, modified=None):
+        """Invoke function withot parameters to get empty changes template"""
         return {
-            'new_fields': changes.get('removed_fields', []),
-            'removed_fields': changes.get('new_fields', []),
-            'modified_fields': {
-                v: k for k, v in changes.get('modified_fields', {}).items()
-            }
+            NEW_FIELDS_KEY: new or [],
+            RM_FIELDS_KEY: removed or [],
+            MOD_FIELDS_KEY: modified or {},
         }
 
-    def convert_changes_to_decisions(self, changes):
+    @classmethod
+    def reverse_changes(cls, changes):
+        return cls.construct_changes(**{
```

Comment:
use `cls.construct_changes(new=..., removed=...)` here instead of **{} notation

Reply:
Done